citations-impl/impl/src/java/org/sakaiproject/citation/impl/NestedCitationValidator.java

```diff
@@ -0,0 +1,88 @@
+package org.sakaiproject.citation.impl;
+
+import org.sakaiproject.citation.api.CitationCollectionOrder;
+import org.sakaiproject.citation.api.CitationValidator;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by nickwilson on 9/29/15.
+ */
+public class NestedCitationValidator implements CitationValidator {
+
+	enum NESTED_CITATION_LIST {
+
+		TOP_LEVEL {
+			@Override
+			public CitationCollectionOrder.SectionType[] getAllowableTypes() {
+				return new CitationCollectionOrder.SectionType[]{
+						CitationCollectionOrder.SectionType.HEADING1,
+						CitationCollectionOrder.SectionType.DESCRIPTION,
+						CitationCollectionOrder.SectionType.CITATION,
+						null
```

Comment:
Could do with a comment as to why null is included. (I read it to allow items that aren't part of the standard sorted citation).